pycbc/mchirp_area.py

```diff
@@ -64,16 +65,30 @@ def insert_args(parser):
 
 
 def from_cli(args):
-    return {'mass_limits': {'max_m1': args.src_class_mass_range[1],
-                            'min_m2': args.src_class_mass_range[0]},
-            'mass_bdary': {'ns_max': args.src_class_mass_gap[0],
-                           'gap_max': args.src_class_mass_gap[1]},
+    if args.src_class_mass_gap_max:
+        return {'mass_limits': {'max_m1': args.src_class_mass_ranges[1],
+                                'min_m2': args.src_class_mass_ranges[0]},
+                'mass_bdary': {'ns_max': args.src_class_mass_ranges[2],
```

Comment:
Here you are assuming that ns max is the 3rd argument to the mass ranges option, but the default has it as 2nd argument ..